packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1806,6 +1820,27 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     _cancelActivePointers();
   }
 
+  /// Method which sends the route name changes over System Channels.
```

Comment:
Maybe better: 'Notifies the platform of a route change."

Reply:
Thanks looks like I totally forgot to replace this one after doing the latest changes!

Reply:
Done.